js/app/widgets/eknWebview.js

```diff
@@ -8,10 +8,13 @@ const GObject = imports.gi.GObject;
 const Gtk = imports.gi.Gtk;
 const WebKit2 = imports.gi.WebKit2;
 
+const Actions = imports.app.actions;
```

Comment:
I think would be better to move this out of the widget and put it in the Document module.

Reply:
yes! Done!